docs/guides/generate-provider/generate_backend.tm.hcl

```diff
@@ -0,0 +1,19 @@
+generate_hcl "_terramate_generated_backend.tf" {
+  condition = tm_try(global.terraform.backend.enabled, true)
+
+  lets {
+    key = "terraform/stacks/by-id/${terramate.stack.id}/terraform.tfstate"
+  }
+
```

Comment:
as this is hardcoded anyway, we can remove the let's and add it in the `content` block.